### DISCUSSION TITLE
Rename the "Gutenberg" WordPress plugin to "Gutenberg Beta"

### DISCUSSION THREAD

**Discussion by adamziel** [WordPress/gutenberg#41789] [General]
The Gutenberg plugin has a two-star rating in the plugin directory: https://wordpress.org/plugins/gutenberg/

Anecdotally, I've heard "Do not install Gutenberg, it only has two stars" is being used as a sales pitch to install a paid site builder. Admittedly, having a plugin called Gutenberg does sound confusing to me, as if Gutenberg was something to be installed and not something that is shipped as a part of WordPress core. It's all explained in the plugin description, but I don't believe many people read it in full.

"Gutenberg Beta" would communicate the plugin's intention more clearly. What do you think?

cc @mtias @scruffian @peterwilsoncc @carolinan @noisysocks @paaljoachim @Mamaduka @jsnajdr @richtabor @aristath @bph

**Comment by Mamaduka** [comment]
cc @annezazu

**Comment by dmsnell** [comment]
Seems fishy to me. My hometown once removed the clocks from the waiting room of public offices because people were complaining about waiting too long.

With undo/redo still broken and a number of basic editing tasks quite buggy, and rapidly changing interfaces that often break existing code, I wonder if we addressed some of the reasons people leave those two stars if our rating wouldn't increase 🤔

**Reply by adamziel** [reply]
@dmsnell oh Gutenberg does have issues, that's for sure. I'm not advocating for hiding the stars, I'm only pointing out there are two different things we refer to as "Gutenberg" and one is shipped with Core while the other is more of an early access.

**Comment by mtias** [comment]
I don't think the plugin itself is Beta. It can include early access features.

**Reply by adamziel** [reply]
Gutenberg Next?

**Comment by paaljoachim** [comment]
I do not feel that ""Gutenberg Beta" would be the correct approach.
As I see the plugin more as a test new block features before these enter into WordPress core. 
There is a fine line between having a beta and a plugin to where one can test out new features.
Perhaps starting with a why the plugin exists and how to use it would be better as it could create a kind of introduction to the plugin. Right now it is very easy to skip the information seen in the description because of the wall of text. Having a few lines with intro text then people could atleast read that before they make a decision in using the plugin or not. Then continue reading if they choose to.

**Comment by peterwilsoncc** [comment]
I think of Gutenberg, the plugin, as the bleeding edge product and the Block Editor, in WordPress, as the stable product.

The two star reviews seem to be a mix of two things: some people hitting bugs because the plugin is bleeding edge/beta/next and more people who simply don't like the Block Editor. The classic editor is mentioned a lot by the latter.

Unfortunately, I don't think renaming the plugin would change that.

It's also worth noting that without closing the `gutenberg` slug and moving to a new slug, the existing reviews would remain linked to the new slug. I suspect doing that due to the rating would be seen as a controversial decision. Once the next version of WordPress is released, it would also use `gutenberg` users on an older version than in the stable plugin.

I know it is disheartening but I think those of us in maintainer roles for Gutenberg or WordPress are stuck with it.

**Comment by noisysocks** [comment]
I always think "WordPress Next" would be a more descriptive name! 😛

But I don't think a new name will help us improve the star rating. I think good old fashioned listening to feedback and fixing bugs is the best path forward there.

**Comment by adamziel** [comment]
I may have miscommunicated my intentions here!

The star rating won’t improve by a mere name change. That’s not my goal.

The perception that Gutenberg = the plugin with early access features could change. Ideally, it would be clear that Gutenberg is part of core and the Plugin is a bleeding edge preview. That’s what I’m hoping for.

**Comment by aristath** [comment]
It is clearly stated in the plugin's page:

> Are you a tech-savvy early adopter who likes testing bleeding-edge and experimental features, and isn’t afraid to tinker with features that are still in active development? If so, this beta plugin gives you access to the latest Gutenberg features for block and full site editing, as well as a peek into what’s to come.

So I don't think we really need to change anything... The plugin is a preview of experimental features that are still in active development. What else is there to say?

Besides, do the star-ratings really affect anything? We go through reported issues and feedback, and fix the things that can be fixed. Most of those reviews should be support requests, and it is generally known and accepted that the rating system is flawed.
In the beginning people were posting negative reviews because they did not want Gutenberg in Core. That trend continues from people who don't understand the nature of the plugin - and that's fine. We can't force people to read the description of the plugin.

IMO the rating in the w.org repo is irrelevant and not something that we should worry about.

**Reply by mtias** [reply]
Also reviews are going to be one of the easiest places where people can voice their opinions and frustrations on the project, what they like / dislike, etc, which is entirely fine. I'm glad there's a place where people can do that.

**Comment by annezazu** [comment]
I don't think anything needs to be changed either. Various efforts have been made to clearly [state intentions of the plugin from News posts](https://wordpress.org/news/2021/04/become-an-early-adopter-with-the-gutenberg-plugin/), to updated descriptions, etc. 

Personally, I'd rather see more of an emphasis on building on the Experiments page to make the experimental elements both more distinct and engaging: https://github.com/WordPress/gutenberg/issues/30922 This will help folks find those experimental features and be aware that they exist within the plugin. 

TLDR: I'd rather we go through other pathways before pursuing a rename, which feels fairly drastic.

**Comment by adamziel** [comment]
Oki, the consensus is clearly that the name change isn't the right measure here. Let's wrap up this thread then – thank you for a productive discussion!

**Comment by jeffpaul** [comment]
It sounds like I'm in the minority here, but I think the renaming might be a worthwhile approach.  I would recommend keeping the slug the same because I see no reason to change that (with all the user impact there) since many folks would be more familiar with the plugin name than the slug (in cases where they're different, see: Yoast).  Even though we may have messaged and marketed and educated and called out in the readme, documentation, and within the plugin experience itself I see no reason to not consider a plugin name update (e.g., Block Editor Beta) to be more precise about what is in the plugin (e.g., things that may or may not eventually make it into WordPress core).

**Reply by imppresser** [reply]
I don't think you're in the minority, you're the developers who really listen to the users, and the users are the majority.
